src/bc_handling/src/bc_handling_base.C

```diff
@@ -321,7 +321,7 @@ namespace GRINS
                                       const std::string& bc_vars,
                                       const std::string& bc_value,
 				      const GetPot& input )
-  {
+  {    
```

Comment:
Can someone do an interactive rebase to remove the "do nothing but screw up the whitespace" parts of the diffs?  ;-)